galsim/utilities.py

```diff
@@ -1485,6 +1464,34 @@ def __init__(self, f):
         self.f = f.__func__
         self.c = weakref.ref(f.__self__)
     def __call__(self, *args):
-        if self.c() is None :
+        if self.c() is None :  # pragma: no cover
             raise TypeError('Method called on dead object')
         return self.f(self.c(), *args)
+
+def EnsureDir(target):
```

Comment:
Is there a reason this is CamelCase?

Reply:
Not particularly.  I just moved it wholesale from where it was in config/output.py since I thought it was more generally useful.  The style there is mostly CamelCase.